codelabs/codelab-02/README.md

```diff
@@ -473,7 +473,7 @@ Submit a screenshot of the front page of the STICs site hosted on your S3. Make
 
 ### Submission
 
-You will submit a zipped directory containing your script, `upload.py`, plus an updated `requirements.txt` if you installed any packages. You will also include the screenshot of the STICs homepage and your audit log.
+You will submit a zipped directory containing your script, `upload.py`, plus a  `pipfile` since you used magic. You will also include the screenshot of the STICs homepage and your audit log.
```

Comment:
"You will submit a zipped directory containing your script, `upload.py`, along with your updated `Pipfile`. The latter should include a dependency on `python-magic` (and any other packages, if any, that you used for this assignment). You will also need to include the screenshot of the STICs homepage and your audit log."